rom/dev/tests/rom_integration_tests/test_cpu_fault.rs

```diff
@@ -0,0 +1,59 @@
+// Licensed under the Apache-2.0 license
+
+use caliptra_builder::firmware;
+use caliptra_hw_model::{BootParams, HwModel, InitParams};
+use elf::{endian::LittleEndian, ElfBytes};
+
+#[test]
+fn test_cpu_fault() {
+    const GLOBAL_EXCEPTION: u32 = 0x01050002;
+
+    let rom_fwid = firmware::rom_from_env();
+    let mut rom = caliptra_builder::build_firmware_rom(rom_fwid).unwrap();
```

Comment:
Instead of calling build_firmware rom, you can call `caliptra_builder::elf2rom(&elf)` after you call `build_firmware_elf`. It'll probably make the test run a little faster.

Reply:
Done